java/recursion/Factorial.java

```diff
@@ -0,0 +1,40 @@
+
+
+import java.util.Scanner;
+
+/**
+ * Created by buddhimah on 10/16/2017.
+ */
+public class Factorial {
+
+    public static  int fac(int num){
+
+        int r;
+
+        if(num==1){
```

Comment:
Handle negative cases by raising an error or by printing that factorial for negative numbers is undefined.